docs/3.0rc/deploy/infrastructure-examples/managed.mdx

```diff
@@ -119,8 +106,8 @@ You can view your compute hours quota usage on the **Work Pools** page in the UI
 
 ## Next steps
 
-Read more about creating deployments in the [deployment guide](/3.0rc/deploy/control-runs/prefect-deploy/).
+Read more about creating deployments in the [deployment guide](/3.0rc/deploy/infrastructure-examples/docker/).
```

Comment:
We probably don't want to call this the deployment guide.

Reply:
```suggestion
Read more about creating deployments in [Run flows in Docker containers](/3.0rc/deploy/infrastructure-examples/docker/).
```